docs/dev/table/connect.md

```diff
@@ -247,10 +247,8 @@ tables:
       topic: test-input
       startup-mode: earliest-offset
       properties:
-        - key: zookeeper.connect
-          value: localhost:2181
-        - key: bootstrap.servers
-          value: localhost:9092
+        zookeeper.connect: localhost:2181,localhost:2182
```

Comment:
why repeat?

Reply:
specific multi hosts format here, no repeat，they have different port

Reply:
@JingsongLi thanks for your review very much.
I addressed your left comments，could you have a more look?

Reply:
I have some confusion. Do we need to start two Kafka processes on one machine?
Is it OK to keep the previous example?

Reply:
Just to specify multi hosts format in a string here,
 maybe 'hostname1:2181,host_name2:2181' will better, how do you think of ?

Reply:
I'm fine with keeping previous example, it is a kafka properties, the Kafka users should know or refer Kafka docs for multiple hosts.

Reply:
but in FLIP 86, we defined an example as following:
`'connector.properties.zookeeper.connect'='localhost:2181,localhost:2182',
'connector.properties.bootstrap.servers'='localhost:9092,localhost:9093',
'connector.properties.group.id'='testGroup'
`
so I think keep this or use 'hostname1:2181,host_name2:2181' can make sense both. @JingsongLi